addon/services/intl.d.ts

```diff
@@ -28,8 +28,6 @@ export default class IntlService extends Service {
   /**
    * Returns an array of registered locale names
    *
-   * @property locales
-   * @public
```

Comment:
@buschtoens `ember-cli-addon-docs@v4` wouldn't run at the very beginning (commit 1).

<details>
<summary>Error log</summary>

ERROR Summary:

  - broccoliBuilderErrorStack: TypeError: Cannot read property 'fields' of undefined
    at generateYuiDocJsonApi (/node_modules/ember-cli-addon-docs-yuidoc/lib/preprocessors/generate-yuidoc-jsonapi.js:78:13)
    at DocsGenerator.build (/node_modules/ember-cli-addon-docs-yuidoc/lib/broccoli/generator.js:23:16)
    at /node_modules/broccoli-caching-writer/index.js:149:21
    at tryCatch (/node_modules/broccoli-caching-writer/node_modules/rsvp/dist/rsvp.js:525:12)
    at invokeCallback (/node_modules/broccoli-caching-writer/node_modules/rsvp/dist/rsvp.js:538:13)
    at publish (/node_modules/broccoli-caching-writer/node_modules/rsvp/dist/rsvp.js:508:7)
    at flush (/node_modules/broccoli-caching-writer/node_modules/rsvp/dist/rsvp.js:2415:5)
    at processTicksAndRejections (internal/process/task_queues.js:77:11)
  - code: [undefined]
  - codeFrame: Cannot read property 'fields' of undefined
  - errorMessage: Cannot read property 'fields' of undefined
        at DocsGenerator
-~- created here: -~-
    at DocsGenerator.Plugin (/node_modules/broccoli-plugin/index.js:7:31)
    at new CachingWriter (/node_modules/broccoli-caching-writer/index.js:18:10)
    at new DocsGenerator (/node_modules/ember-cli-addon-docs-yuidoc/lib/broccoli/generator.js:15:5)
    at Class.createDocsGenerator (/node_modules/ember-cli-addon-docs-yuidoc/index.js:9:12)
    at /node_modules/ember-cli-addon-docs/lib/models/plugin-registry.js:32:11
    at Array.map (<anonymous>)
    at PluginRegistry.createDocsGenerators (/node_modules/ember-cli-addon-docs/lib/models/plugin-registry.js:31:43)
    at Class.postprocessTree (/node_modules/ember-cli-addon-docs/index.js:302:43)
    at /node_modules/ember-cli/lib/utilities/addon-process-tree.js:6:25
    at Array.reduce (<anonymous>)
-~- (end) -~-
  - errorType: Build Error
  - location:
    - column: [undefined]
    - file: [undefined]
    - line: [undefined]
    - treeDir: [undefined]
  - message: Cannot read property 'fields' of undefined
        at DocsGenerator
-~- created here: -~-
    at DocsGenerator.Plugin (/node_modules/broccoli-plugin/index.js:7:31)
    at new CachingWriter (/node_modules/broccoli-caching-writer/index.js:18:10)
    at new DocsGenerator (/node_modules/ember-cli-addon-docs-yuidoc/lib/broccoli/generator.js:15:5)
    at Class.createDocsGenerator (/node_modules/ember-cli-addon-docs-yuidoc/index.js:9:12)
    at /node_modules/ember-cli-addon-docs/lib/models/plugin-registry.js:32:11
    at Array.map (<anonymous>)
    at PluginRegistry.createDocsGenerators (/node_modules/ember-cli-addon-docs/lib/models/plugin-registry.js:31:43)
    at Class.postprocessTree (/node_modules/ember-cli-addon-docs/index.js:302:43)
    at /node_modules/ember-cli/lib/utilities/addon-process-tree.js:6:25
    at Array.reduce (<anonymous>)
-~- (end) -~-
  - name: Error
  - nodeAnnotation: [undefined]
  - nodeName: DocsGenerator
  - originalErrorMessage: Cannot read property 'fields' of undefined
  - stack: TypeError: Cannot read property 'fields' of undefined
    at generateYuiDocJsonApi (/node_modules/ember-cli-addon-docs-yuidoc/lib/preprocessors/generate-yuidoc-jsonapi.js:78:13)
    at DocsGenerator.build (/node_modules/ember-cli-addon-docs-yuidoc/lib/broccoli/generator.js:23:16)
    at /node_modules/broccoli-caching-writer/index.js:149:21
    at tryCatch (/node_modules/broccoli-caching-writer/node_modules/rsvp/dist/rsvp.js:525:12)
    at invokeCallback (/node_modules/broccoli-caching-writer/node_modules/rsvp/dist/rsvp.js:538:13)
    at publish (/node_modules/broccoli-caching-writer/node_modules/rsvp/dist/rsvp.js:508:7)
    at flush (/node_modules/broccoli-caching-writer/node_modules/rsvp/dist/rsvp.js:2415:5)
    at processTicksAndRejections (internal/process/task_queues.js:77:11)

</details>

By adding `console.log` and checking when the loop terminates due to an error,

```js
// node_modules/ember-cli-addon-docs-yuidoc/lib/preprocessors/generate-yuidoc-jsonapi.js

module.exports = function generateYuiDocJsonApi(inputPaths, project) {
  let document = generateYuiDoc(inputPaths, project);

  let modules = {};
  let klasses = {};

  ...

  for (let item of document.classitems) {
    let module = modules[item.file];
    let klass = klasses[item.class];

    console.log(item.file);

    ...
  }
}
```

I found that `addon/services/intl.d.ts` and `addon/services/intl.js` were causing some issue. Deleting the lines with `@property` and `@public` helped with running the demo app, but I'm not sure what the right approach should have been.

Do we need these comments for documentation? If so, how could we reintroduce them without breaking the demo app?